src/CabanaPD_Boundary.hpp

```diff
@@ -223,13 +213,13 @@ struct BoundaryCondition<BCIndexSpace, ForceUpdateBCTag>
             "CabanaPD::BC::apply", policy, KOKKOS_LAMBDA( const int b ) {
                 auto pid = index_space( b );
                 for ( int d = 0; d < 3; d++ )
-                    f( pid, d ) += value;
+                    f( pid, d ) = value;
```

Comment:
@streeve : Why is the "+" removed?

Reply:
This is only because of how the diff shows up. See below where the crack branch BC is replaced by the update BC